### PR TITLE
Update AndroidManifest.xml

### PR DESCRIPTION
**Pull request by uleliaa**

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -25,4 +25,5 @@
         </activity>
     </application>
 
+
 </manifest>
```
Comment (uleliaa) [inline app/src/main/AndroidManifest.xml:29]: Yea

Comment (uleliaa) [inline app/src/main/AndroidManifest.xml:29]: app/src/main/AndroidManifest.xml

**Review by uleliaa — COMMENTED** [review]
Approve

**Review by uleliaa — COMMENTED** [review]
Approve

**Comment by tatopane** [comment]
Please don't spam the project. Thanks

**Pull request closed without merging**